pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
@wongma7 isn't this how we handle GIDs today to circumvent setting the GID on the PVC which doesn't have access to the actual physical asset? 

I guess what I am asking is, this isn't a new field, right? This was used in the past for the GID communication issue between pv/pvc?

Please clarify